provider/terraform.rb

```diff
@@ -135,62 +136,68 @@ def titlelize_property(property)
     # per resource. The resource.erb template forms the basis of a single
     # GCP Resource on Terraform.
     def generate_resource(data)
-      # TODO: Azure switch
-      azure_generate_resource data
-
-      # dir = data.version == 'beta' ? 'google-beta' : 'google'
-      # target_folder = File.join(data.output_folder, dir)
-
-      # name = data.object.name.underscore
-      # product_name = data.product.name.underscore
-      # filepath = File.join(target_folder, "resource_#{product_name}_#{name}.go")
-
-      # data.generate('templates/terraform/resource.erb', filepath, self)
-      # generate_documentation(data)
+      # Multi Cloud Swith
+      if $cloud == 'MAC'
```

Comment:
Similar comment as before. We might be able to minimize the changes to one line.

Reply:
Similarly, resolved.